src/modules/client/pages/ni/billing/ManualBills.vue

```diff
@@ -1,6 +1,30 @@
 <template>
   <q-page class="client-background q-pb-xl">
     <ni-title-header title="Factures manuelles" padding />
+     <ni-simple-table :columns="columns" :pagination.sync="pagination" :data="manualBills"
+      :loading="tableLoading" data-cy="manual-bills-table">
```

Comment:
tu n'as pas besoin du data-cy c'est pour les tests end-to-end

Reply:
done